ToolingVersions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE-CODE in the project root for license information. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
-    <TargetFramework>net8.0-windows10.0.22000.0</TargetFramework>
+    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>

```

Comment:
For anyone looking for the dependencies for the segmented tool:

https://www.nuget.org/packages/CommunityToolkit.WinUI.Controls.Segmented/8.1.240328-rc